docs/import-from-whitehall.md

```diff
@@ -0,0 +1,136 @@
+1. Navigate to https://signon.integration.publishing.service.gov.uk/api_users
+1. Click 'Create API user'
+1. Enter a name (e.g. your name followed by 'Whitehall Import to Content Publisher Migration Testing') and an email address
+1. Once the user is created, click 'Add application token'
+1. Select 'Whitehall', then add the 'Export data' permission
+1. Copy the Whitehall token - you will not be able to access this again
+1. Click 'Add application token' again
+1. Select 'Publishing API'
+1. Copy the Publishing API token - you will not be able to access this again
+
+These tokens can then be set as environment variables:
+
+```
+export whitehall_bearer_token=<>
+export publishing_api_bearer_token=<>
+```
+
+Replace `<>` with the bearer tokens you have previously obtained.
+
+### Overriding Plek URLs
+
+By default, Plek will point to your local development environment when running rake tasks in govuk-docker.  Therefore, you will need to override the Plek URLs.
+
+For integration:
+
+```
+export publishing_api_url=https://publishing-api.blue.integration.govuk.digital
+export whitehall_url=https://whitehall-admin.integration.publishing.service.gov.uk
+```
+
+For staging:
+
+```
+export publishing_api_url=https://publishing-api.staging.publishing.service.gov.uk
+export whitehall_url=https://whitehall-admin.staging.publishing.service.gov.uk
+```
+
+These environment variables will be passed into govuk-docker in later steps.
+
+### Setting up Sidekiq
+
+In order to run a migration locally, Sidekiq must be running when the rake tasks are executed.  As jobs are run independently of the rake task, Plek overrides and bearer tokens are also required here.
+
+```
+govuk-docker run -e PLEK_SERVICE_PUBLISHING_API_URI=$publishing_api_url -e PUBLISHING_API_BEARER_TOKEN=$publishing_api_bearer_token -e PLEK_SERVICE_PUBLISHING_API_URI=$publishing_api_urlPLEK_SERVICE_WHITEHALL_ADMIN_URI=$whitehall_url -e WHITEHALL_BEARER_TOKEN=$whitehall_bearer_token content-publisher-worker bundle exec sidekiq -C config/sidekiq.yml
+```
+
+### Running a migration
+
+Either a single document or multiple document import can be run, as detailed in previous sections.
+
+For example:
+
+```
+govuk-docker run -e PLEK_SERVICE_PUBLISHING_API_URI=$publishing_api_url -e PUBLISHING_API_BEARER_TOKEN=$publishing_api_bearer_token -e PLEK_SERVICE_PUBLISHING_API_URI=$publishing_api_url -e PLEK_SERVICE_WHITEHALL_ADMIN_URI=$whitehall_url -e WHITEHALL_BEARER_TOKEN=$whitehall_bearer_token content-publisher-lite bundle exec rake import:whitehall_migration["nuclear-decommissioning-authority","news_article"]
+```
+
+In order to access the URL printed at the end of the rake task, the Content Publisher application will need to be running locally:
+
+```
+govuk-docker up content-publisher-app
+```
+
+### Accessing a Rails console
+
+You may wish to open a Rails console that allows you to perform tasks on Whitehall documents (e.g. unlock or lock a document in order to test the import task multiple times).
```

Comment:
You might want to link to https://github.com/alphagov/whitehall/blob/master/docs/migration_to_content_publisher/locked-documents.md for the "unlock or lock documents" since that is potentially unfamiliar terminology.

Reply:
Added a link to this doc